OrbitGrpcProtos/capture.proto

```diff
@@ -177,6 +177,50 @@ message AddressInfo {
   }
 }
 
+message GpuQueueSubmission {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  repeated GpuSubmitInfo submit_infos = 2;
+  repeated GpuDebugMarker completed_markers = 3;
+  uint32 num_begin_markers = 4;
```

Comment:
This should probably be `int32` unless there's a specific reason for it to be unsigned.

Reply:
Done.